### PR TITLE
Add 2019 models to the gallery.

### PR DESCRIPTION
**Pull request by schmmd**
With these additions, we're at 11 models without a full-time AI2 employee in the authors.

### CODE REVIEW

File: _data/gallery.yml [hunk 1]
```diff
@@ -1,3 +1,92 @@
+- title:    "Be Consistent! Improving Procedural Text Comprehensionusing Label Consistency"
```
Comment (matt-gardner) [inline _data/gallery.yml]: `s/Comprehensionusing/Comprehension using/`

File: _data/gallery.yml [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+  authors:  "Xinya Du, Bhavana Dalvi Mishra, Niket Tandon, Antoine Bosselut, Wen-tau Yih, Peter Clark, Claire Cardie"
+  venue:    "NAACL 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Be-Consistent-!-Improving-Procedural-Text-using-Du-Mishra/6a4b27f8bde3e8cd65f4d9633339f7149b878672"
+  - code:   "https://github.com/allenai/propara"
+
+###############################################################################
+- title:    "Structural Scaffolds for Citation Intent Classification in Scientific Publications"
+  authors:  "Arman Cohan, Waleed Ammar, Madeleine van Zuylen, Field Cady"
+  venue:    "ArXiv 2019"
```
Comment (matt-gardner) [inline _data/gallery.yml]: You have arxiv 2019 listed for all of the below, but some of them (maybe all, or most) were published at conferences.  We need to find the right venue for these, or it looks bad.

File: _data/gallery.yml [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+###############################################################################
+- title:    "Combining Distant and Direct Supervision for Neural Relation Extraction"
+  authors:  "Iz Beltagy, Kyle Lo, Waleed Ammar"
+  venue:    "ArXiv 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Combining-Distant-and-Direct-Supervision-for-Neural-Beltagy-Lo/8ed76934cf262f38aa9fa0d937c2dcd4d2673684"
+  - code:   "https://github.com/allenai/comb_dist_direct_relex/"
+
+###############################################################################
+- title:    "Still a Pain in the Neck:Evaluating Text Representations on Lexical Composition"
```
Comment (matt-gardner) [inline _data/gallery.yml]: `s/Neck:Evaluating/Neck: Evaluating/`

File: _data/gallery.yml [hunk 4]
```diff
@@ -1,3 +1,92 @@
...
+###############################################################################
+- title:    "75 Languages, 1 Model: Parsing Universal Dependencies Universally"
+  authors:  "Daniel Kondratyuk"
+  venue:    "ArXiv 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/75-Languages%2C-1-Model%3A-Parsing-Universal-Kondratyuk/6a2daaa16306c8b06d7711e55ffe9ca08dd68c7a"
+  - code:   "https://github.com/hyperparticle/udify"
+
+###############################################################################
+- title:    "DROP: A Reading Comprehension Benchmark Requiring Discrete Reasoning Over Paragraphs"
```
Comment (matt-gardner) [inline _data/gallery.yml]: This one is in the main repo; not sure why it should be here.

File: _data/gallery.yml (hunk 2 above)
Comment (schmmd) [reply _data/gallery.yml]: @matt-gardner I figured they were prepublished in ArXiv and the conferences would be clear later.  I took this information from Semantic Scholar.  Do you think it should be easy to determine the conference now?

File: _data/gallery.yml (hunk 4 above)
Comment (schmmd) [reply _data/gallery.yml]: I removed DROP.  I would include models that are in the main repo if they are contributed.

**Comment by schmmd** [comment]
I found venues for many of the ArXiv papers.  If you are able to, feel free to add venues for the rest.

**Comment by schmmd** [comment]
No worries--it fell off my radar too.

**Pull request merged**